### DISCUSSION TITLE
How to sort M2O in reverse order

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#11451] [Archive]
Let's take a collection of `Tasks` that has an M2O collection of `Comments`

The comments should be sorted by " most recently updated first ".
In `Relationship`, the `Sort` field does not accept minus `-date_updated`.

How to show the comments in the reverse order of  the field `date_updated`?

_BTW, anyhow it looks like that the Sort field is working only if you choose the same field that the one displayed:
For instance, if the drop-down of_ `n items` _shows the Comments' `Title` and you choose `Title` as the sort field, 
=> OK, the comments' titles will be sorted accordingly to the `Title`.
But, If you choose another field for sorting like `date_updated` or another one, the sorted order appears to be stucked to the `id` field._

**Comment by rijkvanzanten** [comment]
There's currently no way to control the sort order of the one-to-many interface. The `sort` option of the relationship configuration controls what field holds a numeric manual sort order 👍🏻

**Comment by HemboPagi** [comment]
is this still the case?
I have very similar need. I have classicators table with valuses used in other tables. When sortin the classificator columnb, then sorting is done by ID value. It is confusing for the user, as one expects to see text values in alphabetical order.
